service/auth/jwt/jwt_test.go

```diff
@@ -0,0 +1,271 @@
+package jwt
+
+import (
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"monorepa/model"
+	"os"
+	"testing"
+
+	"github.com/golang-jwt/jwt"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCreateUserJWTToken(t *testing.T) {
+	type request struct {
+		userName string
+	}
+
+	type want struct {
+		isError bool
+	}
+
+	type env struct {
+		set   func()
+		unset func()
+	}
+
+	testCases := []struct {
+		name    string
+		request request
+		want    want
+		env     env
+	}{
+		{
+			name: "normal request",
+			request: request{
+				userName: "bob",
+			},
+
+			want: want{
+				isError: false,
+			},
+			env: env{
+				set: func() {
+					os.Setenv("TOKEN_EXPIRE", "10")
```

Comment:
It would be great to provide some kind of config for SUT. This will make SUT independent from env and also tests will not change env.